### ISSUE TITLE
Dense layer activation

### ISSUE THREAD

**Issue by elv-zhiyun** [Yashgh7076/Eluvio-ML-Scene-Segmentation#1]
Hi, I noticed the activation functions of the last dense layer are "softmax" in the convolutional and dense networks. I think "sigmoid" is more appropriate for binary cross entropy loss/binary focal loss. Also that since # of output channels is 1 in the last layer, using "softmax" here should convert all the predictions to 1. Could that be the reason of the loss doesn't decrease?

**Comment by Yashgh7076** [comment]
Hi, I will look into it. I thought that the predictions were all 1 because I was weighing positive examples too heavily (alpha = 9) in the focal loss. Thanks for pointing it out.
